wled00/bus_manager.h

```diff
@@ -6,6 +6,8 @@
  */
 
 #include "const.h"
+#include <array>
```

Comment:
I don't think we ended up using `<array>` in the end.

Reply:
😄 
There are a lot of relics like this.